scanner/services/matcher.go

```diff
@@ -146,3 +148,22 @@ func (s *matcherService) RegisterServiceHandler(_ context.Context, _ *runtime.Se
 	// Currently we do not set up gRPC gateway for the matcher.
 	return nil
 }
+
+func (s *matcherService) notes(ctx context.Context, vr *v4.VulnerabilityReport) []v4.VulnerabilityReport_Note {
+	dists := s.matcher.GetKnownDistributions(ctx)
+	if len(dists) == 0 {
+		return []v4.VulnerabilityReport_Note{v4.VulnerabilityReport_NOTE_OS_VULNERABILITIES_UNAVAILABLE}
+	}
+	dist := vr.Contents.Distributions[0]
```

Comment:
Is `vr.Contents.Distributions` guaranteed to be non-nil and len > 0 here?

Reply:
yeah `len(dists) == 0` captures both `nil` and non-nil with length == 0

Reply:
sorry, I misread this. I will update this